tests/test_dependency_parser.py

```diff
@@ -54,6 +54,9 @@ class TestDependencyParser():
         ('pyp2rpm~=0.9.3.1', True, True,
          [['Requires', 'pyp2rpm', '({name} >= 0.9.3.1 with {name} < 0.9.4)']]
          ),
+        ('nb2plots>0+unknown', True, True,
+         [['Requires', 'nb2plots', '{name} > 0']]
```

Comment:
Shouldn't this be `>= 0`, if the `+unknown` part is ignored in generators?
Or maybe just drop the Version restriction and require only plain `nb2plots`, since all versions (except 0~foo) satisfy `>= 0`?

Reply:
Since "+unknown", the original requirement is simplified to "nb2plots>0", and the function output is "nb2plots > 0".  

IIRC, this requirement was used in a package, which is how I noticed that the conversion routines crash if the version required is all zeros.

Reply:
Okay, if this works for real world examples, then LGTM.